source/io/Pipe.ooc

```diff
@@ -56,14 +58,14 @@ Pipe: abstract class {
 		PipeWriter new(this)
 	}
 	new: static func -> This {
-		version(unix || apple) {
-			return PipeUnix new() as This
-		}
-		version(windows) {
-			return PipeWin32 new() as This
-		}
-		Exception new(This, "Unsupported platform!\n") throw()
-		null
+		result: This = null
+		version(unix || apple)
+			result = PipeUnix new() as This
```

Comment:
I think this can generate warnings in coverity about not using old value of `result` before writing new. But we can solve this later.